aries_vcx/src/handlers/issuance/issuer.rs

```diff
@@ -164,6 +165,23 @@ impl Issuer {
         .await
     }
 
+    pub async fn send_revocation_notification(&mut self, comment: Option<String>, send_message: SendClosure) -> VcxResult<()> {
+        // TODO: Check if actually revoked
+        if self.issuer_sm.is_revokable() {
+            let rev_msg = RevocationNotification::create()
+                .set_credential_id(self.get_rev_reg_id()?, self.get_rev_id()?)
+                .set_comment(comment)
+                .set_thread_id(&self.get_thread_id()?)
```

Comment:
As per private discussion, I think this message should not have `thread_id`, only `@id`